R/nma_data.R

```diff
@@ -1850,3 +1850,54 @@ nfactor <- function(x, ..., numeric = TRUE, resort = FALSE) {
     return(factor(x, levels = stringr::str_sort(unique(x), numeric = numeric), ...))
   }
 }
+
+#' Calculate baseline risk
+#'
+#' @param network An `nma_data` object.
+#' @param link The name of the link function to use.
+#' @param treatment The reference treatment. If `NULL` (the default), use the `network`
+#'   reference treatment.
+#'
+#' @return The baseline risk as a single numeric value.
+#' @export
```

Comment:
```suggestion
#' @noRd
```

I don't think this function should be exported. The calculation is only crude, and there is no need for the user to call it.